Traceback (most recent call last):
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\larsenyg\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
%matplotlib inline

# Itererbart intervall for psi fra -1 til 1
psi_interval = np.linspace( -1 , 1 , 100 )

# aspektforhold plate
s_over_l = 0.25

C_e0 = (1 + ( s_over_l )**2 )**2 * ((2.1) / ( 0 + 1.1))

x_verdier = []
y_verdier = []

for psi in psi_interval: 
    # Ved strekk 
    if psi < 0:
        if s_over_l**(-1) <= (3/2 ): 
            C_f = 24 * (s_over_l)**2 

        else :
            C_f = 2 + 16 * ( s_over_l )**2 + 8 * ( s_over_l ) ** 4 
        
        bulekoeffisient = ( 1 + abs(psi) ) * C_e0 - abs(psi) * C_f + 10 * abs(psi) * ( 1 + abs(psi) ) + ( s_over_l )**2 

#bulekoeffisient = ( 1 + psi ) * C_e0 - psi * C_f + 10 * psi * ( 1 + psi ) + ( s_over_l )**2 
        
    # Ingen strekk
    else:
        bulekoeffisient = (1 + ( s_over_l )**2 )**2 * ((2.1) / ( psi + 1.1))
        
    # Lagrer i lister for plot
    x_verdier.append(psi) 
    y_verdier.append(bulekoeffisient)


# Plotter

plt.suptitle('Bulekoeffisient der spenningsretning gÃ¥r pÃ¥ tvers av stiverretning')
plt.ylabel('C [-]')
plt.xlabel('Ïˆ [-]')
plt.plot(x_verdier , y_verdier)
plt.show()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Cell [1;32mIn [4], line 4[0m
[0;32m      1[0m get_ipython()[38;5;241m.[39mrun_line_magic([38;5;124m'[39m[38;5;124mmatplotlib[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124minline[39m[38;5;124m'[39m)
[0;32m      3[0m [38;5;66;03m# Itererbart intervall for psi fra -1 til 1[39;00m
[1;32m----> 4[0m psi_interval [38;5;241m=[39m [43mnp[49m[38;5;241m.[39mlinspace( [38;5;241m-[39m[38;5;241m1[39m , [38;5;241m1[39m , [38;5;241m100[39m )
[0;32m      6[0m [38;5;66;03m# aspektforhold plate[39;00m
[0;32m      7[0m s_over_l [38;5;241m=[39m [38;5;241m0.25[39m

[1;31mNameError[0m: name 'np' is not defined
NameError: name 'np' is not defined

